src/main/java/frc/robot/RobotContainer.java

```diff
@@ -276,7 +279,13 @@ public void populateSysIdDashboard() {
     tab.add("Tur Roll Quasi Rev", turretSubsystem.sysIdRollerQuasistaticCommand(kReverse)).withPosition(columnIndex, 1);
     tab.add("Tur Roll Dynam Fwd", turretSubsystem.sysIdRollerDynamicCommand(kForward)).withPosition(columnIndex, 2);
     tab.add("Tur Roll Dynam Rev", turretSubsystem.sysIdRollerDynamicCommand(kReverse)).withPosition(columnIndex, 3);
-
+  
+    // Testing
+    tab = Shuffleboard.getTab("Testing");
+    
+    tab.addNumber("Number of Tests Run", () -> testCommand.getTestState());
+    tab.add("Start LED Progress Bar", new LEDProgressBarCommand(ledSubsystem, testCommand::getTestState)).withPosition(0, 0);
+    tab.add("Start Testing", testCommand);
```

Comment:
I have one final improvement to suggest. You could run the LEDs and tests in a deadline group:
```
    tab.add("Start Testing", testCommand
        .deadlineWith(new LEDProgressBarCommand(ledSubsystem, testCommand::getTestState)).withName("Test Mode"));
```